src/sentry/api/serializers/models/release.py

```diff
@@ -161,8 +158,7 @@ def _get_deploy_metadata(self, item_list, user):
             }
         return result
 
-    def get_attrs(self, item_list, user, *args, **kwargs):
-        project = kwargs.get('project')
+    def __get_release_data_no_environment(self, project, item_list):
```

Comment:
A note here that the project may be `None`[1] would be helpful since the way the method signature is defined doesn't  really imply that.

1: I'm assuming the `if project` below should really be an `if project is None` instead of a boolean coersion.

Reply:
Typically I've used `if object` as a true/false since it'll yield the exact same results as `if project is None` in this case. That said, I'll change it if you prefer it the other way. In the past I've only done `if object is None` when I'm afraid of a false positive (i.e. object could be 0 which evaluates to false but I only want a false returned when the object is None). I think that's just a style preference unless I'm missing something @tkaemming

Reply:
Yeah, it's mainly a stylistic thing and your preferences might differ. I guess that I personally lean towards being more specific when possible, but you're right, the approach here does catch more values (whether or not those values should have been passed as arguments in the first place is another question.) Generally [PEP 8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) recommends explicit `is None` checks when possible, but I think that's context dependent and not a unilaterally applicable rule.